### DISCUSSION TITLE
BTable and stickyHeader

### DISCUSSION THREAD

**Discussion by rubebop** [cdmoro/bootstrap-vue-3#1225] [Q&A]
I'm trying to figure out how to create a **BTable** with a given size with a **stickyHeader**.
In **BV 2** that was pretty convenient as setting the property as follows worked well:
`:sticky-header="availableTableHeight"`,
where `availableTableHeight` was in my case a computed property giving values such as "381px".

In the [documentation of this project](https://bootstrap-vue-next.github.io/bootstrap-vue-next/docs/components/table.html) it seems that the aforementioned property is also available, only as a booleanish value.

If I use that property now: `sticky-header` (i.e. implicitely set as true): I can't observe any effect, why?
Even if I observed an effect, how could I set a given **length**?  If set a certain length on that element or a parent element (such as a div), the table is just as long as the data.

Thanks a lot in advance.

**Comment by oec-himal** [comment]
Any workaround to this problem?

**Reply by VividLemon** [reply]
https://github.com/bootstrap-vue-next/bootstrap-vue-next/commit/f2973bb266c5422407d148cfeed383c5f66263f1
